README.md

```diff
@@ -78,6 +78,9 @@ func main() {
 }
 ```
 
+If you need to customize the GET and DELETE endpoints use
```

Comment:
That's a good question. From having talked with people integrating tusd into their system, the need of using custom headers, e.g. for authentication is presented.

In the future, there will be an easier way to disable upload termination entirely, what may be interesting for your use case.